<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deletion_Widget</class>
 <widget class="QWidget" name="Deletion_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PlusButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="AmountInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="MinusButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Deletion">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLabel" name="AnimationWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>240</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tlwg Typist</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Collapse">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <zorder>AnimationWidget</zorder>
  <zorder>PlusButton</zorder>
  <zorder>AmountInput</zorder>
  <zorder>MinusButton</zorder>
  <zorder>Deletion</zorder>
  <zorder>Collapse</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
